packages/runtime/test/src/interpreter/interpreter.ts

```diff
@@ -45,7 +44,7 @@ describe("Interpreter", function () {
 			txn: TXN_OBJ,
 			txID: TXN_OBJ.txID,
 		});
-		interpreter.tealVersion = 1;
+		interpreter.tealVersion = 6;
```

Comment:
This line should be erase because this one is in `resetInterpreter` but not setting the version of the `interpreter`